draft-ietf-rats-eat.md

```diff
@@ -1683,7 +1683,10 @@ downstream consumer should leverage a communication security protocol
 
 However, assume the EAT of the previous example is hierarchical and
 each claim subset for a downstream consumer is created in the form of
-a nested EAT.  Then, Transport Layer Security between the receiving and
+a nested EAT.  Then the nested EAT is itself cryptographically-verifiable (due to its
+COSE envelope)
```

Comment:
Maybe mention COSE encrypted too since that's in the above example and because the role of TLS here is more about privacy than authentication/integrity.